OCPBUGS-1772: Pre-caching feature blocks CGU CR (#51022)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

+
(link:https://issues.redhat.com/browse/OCPBUGS-753[*OCPBUGSM-753*])

> * Enabling the pre-caching feature of {cgu-operator-full} blocks the `ClusterGroupUpgrades` CR. To workaround this issue, do not enable the pre-caching feature. (link:https://issues.redhat.com/browse/OCPBUGS-1772[*OCPBUGS-1772*])

Reviewer: ```suggestion
* Enabling the pre-caching feature of {cgu-operator-full} blocks the `ClusterGroupUpgrades` custom resource (CR). As a workaround, do not enable the pre-caching feature. (link:https://issues.redhat.com/browse/OCPBUGS-1772[*OCPBUGS-1772*])
```

Author: @opayne1 Thank you! Updated
